src/drivers/code_hosting/github.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+
+function create_pull_request {
+  local repository=$1
+  local branch=$2
+
+  open_browser "https://github.com/$repository/compare/$branch?expand=1"
```

Comment:
We could get away with these repos only defining the function `repository_url` and `pull_request_url`